packages/ckeditor5-table/tests/tableselection.js

```diff
@@ -831,6 +831,120 @@ describe( 'table selection', () => {
 		} );
 	} );
 
+	describe( 'getAnchorCell() and getFocusCell()', () => {
```

Comment:
After refactoring to a getter the proper describe would be:

```javascript
describe( '#anchorCell and #focusCell', ... );
```

Reply:
The other things is that those tests can be split so one set of tests for `#anchor` and another for `#focus`. After reading that cases for anchor and focus are tested separately.

But I see them tangled somehow - so if you don't see a reason - don't split ;)